libraries/ashrae/guideline36/5.16.14-multiple-zone-vav-ahu-afdd.ttl

```diff
@@ -0,0 +1,430 @@
+                sh:qualifiedMinCount 1 ;
+            ] ;
+        sh:qualifiedMinCount 1 ;
+        ] ;
+    ] .
+
+:fc-2 a sh:NodeShape, owl:Class, bmotif:Analytics_Application ;
+    bmotif:domain bmotif:HVAC ;
+    rdfs:label "FC#2" ;
+    rdfs:comment "MAT too low; should be between OAT and RAT" ;
+    sh:class brick:Air_Handling_Unit ;
+    sh:property [
+        rdfs:label "MAT_AVG" ;
+        rdfs:comment "rolling average of mixed air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Mixed_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    sh:property [
+        rdfs:label "RAT_AVG" ;
+        rdfs:comment "rolling average of return air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Return_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    sh:property [
+        rdfs:label "OAT_AVG" ;
+        rdfs:comment "rolling average of outdoor air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Outside_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] .
+
+:fc-3 a sh:NodeShape, owl:Class, bmotif:Analytics_Application ;
+    rdfs:label "FC#3" ;
+    rdfs:comment "MAT too high; should be between OAT and RAT" ;
+    rdfs:subClassOf :fc-2 ;
+.
+
+:fc-4 a sh:NodeShape, owl:Class, bmotif:Analytics_Application ;
+    bmotif:domain bmotif:HVAC ;
+    rdfs:label "FC#4" ;
+    rdfs:comment "Too many changes in Operating State" ;
+    sh:class brick:Air_Handling_Unit ;
+    sh:property [
+        rdfs:label "Delta_OS" ;
+        rdfs:comment "number of changes in OS during the previous 60 minutes (moving window)" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Operating_Mode_Status ] ;
+        sh:qualifiedMinCount 1 ;
+    ] .
+
+:fc-5 a sh:NodeShape, owl:Class, bmotif:Analytics_Application ;
+    bmotif:domain bmotif:HVAC ;
+    rdfs:label "FC#5" ;
+    rdfs:comment "SAT too low; should be higher than MAT" ;
+    sh:class brick:Air_Handling_Unit ;
+    sh:property [
+        rdfs:label "SAT_AVG" ;
+        rdfs:comment "rolling average of supply air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Supply_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    ] ;
+    sh:property [
+        rdfs:label "MAT_AVG" ;
+        rdfs:comment "rolling average of mixed air temperature" ;
+        sh:path brick:hasPoint ;
+        sh:qualifiedValueShape [ sh:class brick:Mixed_Air_Temperature_Sensor ] ;
+        sh:qualifiedMinCount 1 ;
+    # ] ;
+    # sh:property [
+    #     rdfs:label "Delta_T_SF" ;
+    #     rdfs:comment "Temperature rise across supply fan" ;
+    #     sh:path brick:hasPoint ;
+    #     TODO missing from brick
+    #     sh:qualifiedValueShape [ sh:class brick:Air_Differential_Temperature_Sensor ] ;
```

Comment:
@gtfierro FYI.

Reply:
I've seen the DeltaT SF modeled as a constant in G36. I'm not sure if anyone has a commonplace sensor for this. I can ask at ASHRAE meeting to see if anyone has thoughts on how to handle this. We  can definitely add a differential air temp sensor though

Reply:
Yes, this is actually an "internal variable" (constant) so I think we can omit from the shapes. Might still be useful to have in Brick in the future.